src/frontend/src/icp/components/fee/BitcoinFeeContext.svelte

```diff
@@ -38,9 +38,9 @@
 			identity: $authIdentity,
 			amount: parseToken({
 				value: `${amount}`,
-				unitName: $tokenDecimals
+				unitName: token.decimals
```

Comment:
Correct me if I'm wrong, but the behaviour stays completely the same with this PR, the only difference is that now `tokenWithFallBackAsIcToken` is passed from the parent component (`IcTokenWizard`) instead of being used directly from `BitcoinFeeContext`. I think your question goes more into the `ckBTC` -> `BTC` conversion direction, which will continue working as it is right now in `main`.

Reply:
Just double-checked - `tokenWithFallBackAsIcToken` uses `token` from the store under the hood, therefore the logic hasn't been updated with this PR. Since it's something that will be updated soon (when migrating `ckBTC` -> `BTC` to the new flow I'm working on atm), is it fine with you that we don't introduce new tests for the soon-to-be-gone code

Reply:
ah understood! thank you for the explanation!